tests/bench/coremark/coremark_main.c

```diff
@@ -320,8 +342,25 @@ for (i = 0; i < MULTITHREAD; i++)
         core_stop_parallel(&results[i]);
     }
 #else
-    iterate(&results[0]);
+    for (i = 0; i < RUN_CNT; i++) {
+#ifdef CONFIG_COREMARK_SIMPLE_REPORT
+        clock_ticks[cpu_id][i] = get_time_ticks();
 #endif
+        iterate(&results[0]);
+    }
+#ifdef CONFIG_COREMARK_SIMPLE_REPORT
+    clock_ticks[cpu_id][RUN_CNT] = get_time_ticks();
+    ee_printf("cpu run ticks-start - ticks-stop : ticks-diff iterations/sec\n");
+    for (i = 1; i <= RUN_CNT; i++) {
+        CORE_TICKS diff_ticks =  clock_ticks[cpu_id][i] - clock_ticks[cpu_id][i-1];
+        ee_printf("%3d %3d %llu - %llu : %llu %llu\n", cpu_id, i,
+                clock_ticks[cpu_id][i-1], clock_ticks[cpu_id][i], diff_ticks,
+				((unsigned long long)results[0].iterations) * 1000000 / diff_ticks);
+    }
+#endif
```

Comment:
Should a new macro or function be introduced for this logic?

Reply:
> Should a new macro or function be introduced for this logic?

fixed.